### ISSUE TITLE
Dawnguard xedits?

### ISSUE THREAD

**Issue by Gpeschke** [EzioTheDeadPoet/SME-FT-#10]
Apparently according to lexy's there's some 'extra' edits for cleaning dawnguard: https://lexyslotd.com/guide/prerequisites/#dawnguard-esm-requires-manual-xedit-cleaning-in-addition-to-the-automated-cleaning)

Knowing that detail was taken care of would be nice.

**Comment by EzioTheDeadPoet** [comment]
All masters are cleaned and Dawnguard did get a manual cleaning, but thb I have no Idea if the cleaned records (i cleaned two manually iirc) are the same as mentioned in Lexy's guide since I followed someone else's instructions back when I did it. But the chance that the cleaned records are the same as with the Lexy Guide is pretty high. But to be sure I recommend to you check it yourself, since I am currently not at my PC.
